code/controllers/subsystem/SSjobs.dm

```diff
@@ -529,9 +529,12 @@ SUBSYSTEM_DEF(jobs)
 				if(istype(G) && !G.prescription)
 					G.upgrade_prescription()
 					H.update_nearsighted_effects()
-	if(late_arrivals_spawning && !joined_late)
+	if(joined_late || job.admin_only)
+		H.create_log(MISC_LOG, "Spawned as \an [H.dna?.species ? H.dna.species : "Undefined species"] named [H]. [joined_late ? "Joined during the round" : "Roundstart joined"] as job: [rank].")
```

Comment:
With this change, line 545 can be shortened since joined_late will never be true there

Reply:
oops missed that one